index.html

```diff
@@ -0,0 +1,146 @@
+            expertise to make bookings in the world affordable for you.
+          </p>
+        </div>
+        <h2 class="promotion">2021.08.01(THU) ~ 6(FRI)</h2>
+        <p>@Our Anniversary special Offer</p>
+      </section>
+      <section class="program">
+        <h2 class="program-header">Products & Services</h2>
+        <hr class="line" />
+        <div class="program-content">
+          <div class="content">
+            <div class="icon"><i class="fas fa-plane-departure"></i></div>
+            <h2>Flight Reservations</h2>
+            <p>
+              Global Travel offers cost effective flight reservations to all
+              destinations.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon"><i class="fas fa-car-alt"></i></div>
+            <h2>Car Hire</h2>
+            <p>
+              Global Travel is an agent for a number of internationally
+              recognized car rental companies.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon"><i class="fas fa-truck"></i></div>
+            <h2>Delivery service</h2>
+            <p>
+              Global Travel delivers airline tickets and itinerary travel
+              documents, free of charge, to its customers.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon"><i class="fas fa-passport"></i></div>
+            <h2>Visas</h2>
+            <p>
+              Global Travel has a department which offers professional and
+              timeous visa services.
+            </p>
+          </div>
+          <div class="content">
+            <div class="icon"><i class="fas fa-user-tag"></i></div>
+            <h2>Specials</h2>
+            <p>
+              Global Travel communicates the “specials” to the customers
+              whenever the specials are received from the airlines.
+            </p>
+          </div>
+        </div>
+        <button class="program-btn" type="button">
+          Join our Travelers Community
+        </button>
+        <a href="#" class="link">SEE ALL PRODUCTS & SERVICES</a>
+      </section>
+      <section class="featured-speakers">
+        <h3 class="featured-speakers-header">Our customers speak</h3>
+        <hr class="line" />
+        <div id="speakers" class="grid"></div>
+      </section>
+      <section class="partner">
+        <h2 class="partner-header">Partners</h2>
+        <hr class="line" />
+        <div class="partner-logo">
+          <div class="logo-group-1">
+            <p><span class="logo--1 logo-partner">maxbnb</span></p>
+            <p><span class="logo--2 logo-partner">Ryes Airlines</span></p>
+            <p><span class="logo--3 logo-partner">Agglos Holels</span></p>
+          </div>
+          <div class="logo-group-2">
+            <p><span class="logo--4 logo-partner">Kibira Vacation</span></p>
+            <p><span class="logo--5 logo-partner">Ubest</span></p>
```

Comment:
- [ ] Follow the original grey design for the logos, right now breaks the style of your page.

Original design:

![image](https://user-images.githubusercontent.com/60631456/124145101-20f48000-da52-11eb-833a-3bfbfedac92e.png)

Your design:

![image](https://user-images.githubusercontent.com/60631456/124145204-35d11380-da52-11eb-912b-0eb77188599d.png)

Reply:
This is fixed now